packages/esm-patient-search-app/src/patient-search/patient-search.resource.tsx

```diff
@@ -1,7 +1,7 @@
 import { openmrsFetch } from '@openmrs/esm-framework';
 
 export function performPatientSearch(query: string, objectVersion: string, ac: AbortController, includeDead: boolean) {
-  const url = `/ws/rest/v1/patient?q=${query}&v=${objectVersion}&includeDead=${includeDead}`;
+  const url = `/ws/fhir2/R4//Patient?name:contains=<q>&identifier=<q>${objectVersion}&includeDead=${includeDead}`;
```

Comment:
@jwamalwa, only changing the URL works?
The FHIR response is different than the REST one. Have you tried to run the change?